### PR TITLE
Bug 1143084 remove unused fonts from code base

### PR DESCRIPTION
**Pull request by openjck**
Supersedes #3198 with bug numbers in the commit messages.

### CODE REVIEW

File: media/stylus/devderby.styl [hunk 1]
```diff
@@ -161,9 +163,9 @@ transform: rotate(18deg);
 
 /* @Resources *********/
 #resources { clear: both; border-top: 4px dotted #746b60; padding: 20px 40px; }
-#resources h3 { font: bold 1.142em/1 "Carto Bold", sans-serif; text-transform: uppercase; letter-spacing: 2px; }
+#resources h3 { font: bold 1.142em/1, sans-serif; text-transform: uppercase; letter-spacing: 2px; }
```
Comment (darkwing) [inline media/stylus/devderby.styl]: We should be defining a font here, yes?

File: media/stylus/devderby.styl [hunk 2]
```diff
@@ -99,8 +101,8 @@ transform: rotate(18deg);
 .hasJS #current-challenge .block { margin: 0; border: 0; }
 
 #tab-challenge header { min-height: 70px; padding: .25em 0 .25em 70px; margin: 0 0 1em -15px; background: url("../img/devderby/trophy-one-sm.png") no-repeat; }
-#tab-challenge .title { font: 200 40px/.95 "Bebas Neue", "League Gothic", Haettenschweiler, "Arial Narrow", sans-serif; color: #000; margin: 0 0 .15em; }
-#tab-challenge .date { font: bold 1.142em/1 "Carto Bold", sans-serif; text-transform: uppercase; letter-spacing: 4px; color: #000; }
+#tab-challenge .title { font: 200 40px/.95 $heading-font-family; color: #000; margin: 0 0 .15em; }
+#tab-challenge .date { font: bold 1.142em/1, sans-serif; text-transform: uppercase; letter-spacing: 4px; color: #000; }
```
Comment (darkwing) [inline media/stylus/devderby.styl]: Define a font here?

File: media/stylus/devderby.styl [hunk 3]
```diff
@@ -86,7 +88,7 @@ transform: rotate(18deg);
 /* @Current @Challenge *********/
 #current-challenge { border: 0; padding-top: 4em; position: relative; min-height: 300px; }
 #current-challenge .tabs { width: 100%; height: 3em; list-style: none; margin: 0; position: absolute; top: 0; left: 0; border-bottom: 3px solid #000; background: transparent none; }
-#current-challenge .tabs li { border: 0; float: left; margin: 0; padding: 0; text-align: center; color: #333; background: transparent none; font: bold 1em/1 "Carto Bold", sans-serif; text-transform: uppercase; letter-spacing: 2px; }
+#current-challenge .tabs li { border: 0; float: left; margin: 0; padding: 0; text-align: center; color: #333; background: transparent none; font: bold 1em/1 sans-serif; text-transform: uppercase; letter-spacing: 2px; }
```
Comment (darkwing) [inline media/stylus/devderby.styl]: Font?

File: media/stylus/devderby.styl [hunk 4]
```diff
@@ -55,8 +57,8 @@ transform: rotate(18deg);
 #upcoming .second { width: 150px; margin-left: -370px; bottom: 45px; }
 #upcoming .third { width: 150px; margin-left: 20px; bottom: 45px; }
 
-#upcoming .title { text-align: center; margin: 0 0 .15em; font: 200 30px/.95 "Bebas Neue", "League Gothic", Haettenschweiler, "Arial Narrow", sans-serif; color: #000; text-shadow: 0 0 20px #f1e8cf, 0 0 10px #fff, 1px 1px 0 rgba(0,0,0,.1); }
-#upcoming .date { text-align: center; margin: 0; font: bold 1em/1 "Carto Bold", sans-serif; text-transform: uppercase; letter-spacing: 3px; text-shadow: 0 0 10px #f1e8cf, 0 0 5px #fff; }
+#upcoming .title { text-align: center; margin: 0 0 .15em; font: 200 30px/.95 $heading-font-family; color: #000; text-shadow: 0 0 20px #f1e8cf, 0 0 10px #fff, 1px 1px 0 rgba(0,0,0,.1); }
+#upcoming .date { text-align: center; margin: 0; font: bold 1em/1 sans-serif; text-transform: uppercase; letter-spacing: 3px; text-shadow: 0 0 10px #f1e8cf, 0 0 5px #fff; }
```
Comment (darkwing) [inline media/stylus/devderby.styl]: font?

File: media/stylus/devderby.styl [hunk 5]
```diff
@@ -47,7 +49,7 @@ transform: rotate(18deg);
 /* @Upcoming challenges *********/
 #upcoming { border-bottom: 4px dotted #746b60; margin-top: -210px; padding: 0 20px 20px 0; }
 #upcoming:after { content: "."; display: block; clear: both; height: 0; visibility: hidden; }
-#upcoming h1 { padding: .25em 85px; background: url("../img/devderby/upcoming-head.png") 50% .75em no-repeat; color: #e35915; font: bold 1.142em/1 "Carto Bold", sans-serif; text-transform: uppercase; letter-spacing: 3px; text-align: center; text-shadow: 5px 0 20px #fff, -5px 0 20px #fff, 0 0 5px #fff, 1px 1px 0 rgba(0,0,0,.1); }
+#upcoming h1 { padding: .25em 85px; background: url("../img/devderby/upcoming-head.png") 50% .75em no-repeat; color: #e35915; font: bold 1.142em/1 sans-serif; text-transform: uppercase; letter-spacing: 3px; text-align: center; text-shadow: 5px 0 20px #fff, -5px 0 20px #fff, 0 0 5px #fff, 1px 1px 0 rgba(0,0,0,.1); }
```
Comment (darkwing) [inline media/stylus/devderby.styl]: font?

**Comment by davidyezsetz** [comment]
@darkwing I didn't define a font because as far as I understood the dev derby is discontinued and the selectors in question don't exist anymore. (At least I couldn't find them at https://developer.mozilla.org/en/demos/devderby)

**Comment by stephaniehobson** [comment]
@darkwing I am 98% sure that Carto was not displaying anywhere on the site because of the cascade. (cruise around with the network panel open, it never loads).

**Comment by stephaniehobson** [comment]
Doing it :) Thanks @davidyezsetz!

**Pull request merged**